Python/Product/Analysis/PythonAnalyzer.cs

```diff
@@ -179,7 +179,7 @@ public partial class PythonAnalyzer : IGroupableAnalysisProject, IDisposable {
                 _interpreterFactory.NotifyImportNamesChanged();
                 _modules.ReInit();
 
-                await LoadKnownTypesAsync().ConfigureAwait(false);
+                await LoadKnownTypesAsync();
```

Comment:
Any situation where we have no sync context, we also shouldn't have any loaded modules (below), so there won't be an issue. If there are loaded modules, they may be enqueued, which means we have an analysis queue and synchronization context available, so any call that isn't within that context is invalid and needs to be fixed.